### PR TITLE
The python2 -> python3 conversion is more involved where Qt5 is conce…

### PR DESCRIPTION
**Pull request by mcb64**
…rned.

QVariant is generally no longer needed, and in fact causes problems because
automatic type conversions aren't being done.

Also, deleting keys while iterating over a dictionary is now a bad thing.
Go figure.

Also added code to modify a motor configuration from the PVs of a particular
motor.

### CODE REVIEW

**Comment by mcb64** [comment]
The handling of QVariant (the polymorphic wrap-everything type in Qt) has changed in pyqt between python2 and python3.  If python2, you had to have it wrap all of the basic types, but there were implicit conversions from QVariant to the normal types.  In python3, they use QVariant internally when passing things to Qt, but they hide this, so you never have to actually include this type.  The downside is that if you *do* include it, the implicit conversions have gone away and you need to explicitly add code to retrieve the underlying value.

tl;dr - All mentions of QVariant needed to be scrubbed from the code.
    --Mike

________________________________

**Comment by ZLLentz** [comment]
Ok, seems good. I will pull this in.
